eoscostarica/eos-rate
# Show Eden rating

## Show Eden rating
## Fix bug when user update her rating

Resolve #688
Resolve #687

###  What does this PR have?
- show bp eden rating
- fix bug when user do rating

### Steps to test

1. Run app
1. Login whit eden account
1. Selected BP and do rating
2. Go to bp profile and see eden rating stats
